### PR TITLE
Kirstie's updates for April

### PR DESCRIPTION
**Pull request by KirstieJane**
Be warned, there's millions and likely a LOT of joys that are missing!

### CODE REVIEW

File: weekly-updates/Kirstie-Whitaker/Weekly-meetings.md [hunk 1]
```diff
@@ -21,6 +22,174 @@ Notes from Kirstie's personal version of the "weekly meetings" file she asks all
...
+* AM is doing a fantastic job nudging me along without making me feel pressured.
+  * Awesome prep for a couple of our big meetings at the end of the month 🌠
+* Participated in the Tidelift livestream (not a podcast!) and wrote a big old twitter thread about so many thoughts!
+  * 🧵 https://twitter.com/kirstie_j/status/1385707607671857155?s=20
+  * 🎥 https://www.twitch.tv/videos/998271615
+* AB advocated against my original plan and I think that was exactly the right call.
+  * WELL DONE 🌺
+
+### What did you achieve
+
```
Comment (Arielle-Bennett) [inline weekly-updates/Kirstie-Whitaker/Weekly-meetings.md:62]: Just a note to say these are *super useful* for programme reporting! It's made it so easy to go back through and collate details for KW to review :)

File: weekly-updates/Kirstie-Whitaker/Weekly-meetings.md [hunk 2]
```diff
@@ -21,6 +22,174 @@ Notes from Kirstie's personal version of the "weekly meetings" file she asks all
...
+  * Masters degree intern from CRI.
+* Feedback on grant panel reviews.
+  * Late, but ultimately I was quite happy with them.
+  * Got them in GitHub too! 🌟
+* Conversation for the collaboration we're currently negotiating.
+  * One not so constructive, the other more so.
+    Both helping to take us on the right path.
+* Job description for ASG RAM 🐏 submitted to recruitment.
+* Invited AM and MS to the TPS Leadership meetings, so valuable to move those conversations forwards.
+  * Progress - led by AB - on the call for applications!!
```
Comment (Arielle-Bennett) [inline weekly-updates/Kirstie-Whitaker/Weekly-meetings.md:80]: We'll get there!

File: weekly-updates/Kirstie-Whitaker/Weekly-meetings.md [hunk 3]
```diff
@@ -21,6 +22,174 @@ Notes from Kirstie's personal version of the "weekly meetings" file she asks all
...
+* 1:1s w MS, IKG, AB, AM, GA, LC, ME, EK,
+* Lab meetings
+  * Overview of CollabW21 🤝
+  * Ethics form 🦖
+  * Goatherding 🐐
+  * Community management 🌻
+
+### What did you struggle with?
+
+* Onboarding documentation is very poor.
```
Comment (Arielle-Bennett) [inline weekly-updates/Kirstie-Whitaker/Weekly-meetings.md:161]: Lab onboarding documentation or other onboarding documentation?

**Review by Arielle-Bennett — APPROVED** [review]
This is awesome, and these updates are soooo useful in lots of different ways - thank you for making time for them!

File: weekly-updates/Kirstie-Whitaker/Weekly-meetings.md (hunk 3 above)
Comment (KirstieJane) [reply weekly-updates/Kirstie-Whitaker/Weekly-meetings.md:161]: Lab onboarding. Just got to find the time to pull it together _sigh_.

**Pull request merged**